modules/reference/pages/configuration/config-menu.adoc

```diff
@@ -1,3 +1,67 @@
+[[ref.webui.config]]
 = Configuration Menu
 
+Only Configuration Administrators or {productname} Administrators see the menu:Main Menu[Configuration] pages.
 
+Within the configuration pages, you can manage systems via configuration files, and configure channels offering configuration files, and configuration files themselves.
+Centrally-managed files are available to multiple systems; locally-managed files are available to individual systems only.
+
+.Differences of System Types
+[NOTE]
+====
+Configuration Managment is available for both client system types, traditionally managed clients ([Management]) and Salt minions ([Salt]).
+Some traditional features are not suitable for Salt minions, and thus not available and excluded from the {webui}.
+====
+
+
+
+== Configuration Management for Salt
+
+Configuration Management is now enabled for Salt.
+The following matrix provides both supported and unsupported configuration management features.
+
+.Missing {webui} Options
+[IMPORTANT]
+====
+Several {webui} tabs will be missing for Salt Configuration Management.
+These features are not suitable for Salt minions.
+====
+
+.Salt Configuration Management
+[cols="1,1", options="header"]
+|===
+| Configuration Management Features | Salt Support Status
+| Global Configuration Channels     | Supported
+| Deploying Files | Supported
+| Comparing Files | Supported (but the logic is currently inverted)
+| Locally Managed Files | Unsupported
+| Sandbox Files | Unsupported
+| Applying the Highstate | Apply the highstate and configuration channels will be deployed to all subscribed systems.
+| File Import from a Client | Unsupported
+| Configuration Macros | Unsupported
+|===
```

Comment:
I'm sure that in another similar table we managed to replace the words 'supported/unsupported' with a green tick/red cross?